### PR TITLE
Addressed comments given in PR #3935

### PR DESCRIPTION
**Pull request by sghai**
Some comments were given earlier on PR #3935. However PR got closed by other reviewers so just incorporated those comments here. Please take a look if it looks good now.

### CODE REVIEW

**Comment by oshtaier** [comment]
ACK pending test result for `test_positive_update_default_org` test

File: robottelo/ui/discoveredhosts.py [hunk 1]
```diff
@@ -100,32 +100,17 @@ def reboot_host(self, hostname):
                 'Could not find the selected discovered host "{0}"'
                 .format(hostname)
             )
-        drop_locator = locators['discoveredhosts.dropdown']
-        self.click((drop_locator % hostname))
-        reboot_locator = locators['discoveredhosts.reboot']
-        self.click((reboot_locator % hostname), wait_for_ajax=False)
+        self.click((locators['discoveredhosts.dropdown'] % hostname))
```
Comment (abalakh) [inline robottelo/ui/discoveredhosts.py]: These extra brackets are redundant, they're needed only with our old `strategy, value` approach to form a tuple, but as this PR is for 6.3 you can drop them.

File: robottelo/ui/discoveredhosts.py [hunk 2]
```diff
@@ -100,32 +100,17 @@ def reboot_host(self, hostname):
...
+        self.click(
+            (locators['discoveredhosts.reboot'] % hostname),
```
Comment (abalakh) [inline robottelo/ui/discoveredhosts.py]: Same here

File: robottelo/ui/discoveredhosts.py [hunk 3]
```diff
@@ -100,32 +100,17 @@ def reboot_host(self, hostname):
...
-            raise UIError(
-                'Could not find the select_action dropdown for bulk operations'
-            )
-        self.click(select_action_element)
-        if not self.find_element(assign_org_element):
-            raise UIError(
-                'Could not find the org element from select_action dropdown'
-            )
-        self.click(assign_org_element)
+            self.click((locators['discoveredhosts.select_host'] % host))
```
Comment (abalakh) [inline robottelo/ui/discoveredhosts.py]: And here

**Comment by sghai** [comment]
@oshtaier @abalakh : thank you for review. comments have been addressed:

test result:

```console
]$ py.test -v tests/foreman/ui/test_discoveredhost.py::DiscoveryTestCase -k test_positive_update_default_org
================================================================ test session starts =================================================================
platform linux2 -- Python 2.7.12, pytest-3.0.2, py-1.4.31, pluggy-0.3.1 -- /usr/bin/python
cachedir: .cache
rootdir: /home/sghai/QE/robottelo, inifile: 
collected 53 items 

tests/foreman/ui/test_discoveredhost.py::DiscoveryTestCase::test_positive_update_default_org <- robottelo/decorators/__init__.py PASSED
tests/foreman/ui/test_discoveredhost.py::DiscoveryTestCase::test_positive_update_default_org_from_settings <- robottelo/decorators/__init__.py SKIPPED

================================================================ 51 tests deselected =================================================================
================================================ 1 passed, 1 skipped, 51 deselected in 380.93 seconds ================================================
```

**Comment by oshtaier** [comment]
APT

**Pull request merged**